doc/source/contributor/airship.rst

```diff
@@ -0,0 +1,97 @@
+Airship
+=======
+
+Airship is a platform that takes YAML documents and turns them into Clouds.
+Takes helm charts, baremetal, security policies and network description and
+creates a cloud
+
+.. image:: https://airship-treasuremap.readthedocs.io/en/stable/_images/architecture.png
+
+* Airship architecture
+    https://airship-treasuremap.readthedocs.io/en/stable/ (Stable)
+
+    https://airship-treasuremap.readthedocs.io/en/latest/ (Latest)
+
+* Git repository
+    https://opendev.org/airship
+
+* Video
+    https://www.youtube.com/watch?v=0eEisMm9ykg
+
+
+Airship architecture parts
```

Comment:
(nit) All three words should be capitalized

Reply:
makes sense. Capitalized words as requested.